### PR TITLE
Add monitoring targets

### PR DESCRIPTION
**Pull request by omegas27**
Towards https://github.com/giantswarm/giantswarm/issues/11721

### CODE REVIEW

**Comment by mig4** [comment]
Question: could these not be done using a `ServiceMonitor`? Seems like they would be perfect for that.

**Comment by omegas27** [comment]
We actually decided to put everything in scrape configs for now to scrape everything and move everything away to one service generic service monitor once labels are added

File: files/templates/additional-scrape-configs.template.yaml [hunk 1]
```diff
@@ -28,7 +28,11 @@
     replacement: {{ .APIServerURL }}:443
   - source_labels: [__meta_kubernetes_node_name]
     target_label: __metrics_path__
+{{ if eq .ClusterType "control_plane" }}
+    replacement: /api/v1/nodes/${1}:9393/proxy/metrics
```
Comment (omegas27) [inline files/templates/additional-scrape-configs.template.yaml]: why 9393 when the port is 9323? https://github.com/giantswarm/giantnetes-terraform/blob/5982dee3a754a43c5ca25749c86f614a90f18a62/templates/vault.yaml.tmpl#L59

Comment (omegas27) [reply files/templates/additional-scrape-configs.template.yaml]: You changed it already, but the port on the tenant clusters should be 9323 and should only work for clusters running 12.0.0+.
Docker is advising to use 9323 https://docs.docker.com/config/daemon/prometheus/#configure-docker so I updated that value in k8scloudconfig

**Review by JosephSalisbury — APPROVED** [review]

**Comment by TheoBrigitte** [comment]
* cluster v11.5.2 - 0 Unhealthy targets
* cluster v9.3.6 - 2 Unhealthy targets (docker and nginx, expected and tracked down)
* control plane - 3 Unhealthy targets (calico, kube-proxy, docker, expected and tracked down)

**Pull request merged**